.github/workflows/cloudbeat-ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 env:
   CONTAINER_SUFFIX: ${{ github.run_id }}
-  CI_ELASTIC_AGENT_DOCKER_TAG: "8.7.0-SNAPSHOT"
+  CI_ELASTIC_AGENT_DOCKER_TAG: "8.13.0-SNAPSHOT"
```

Comment:
@orouz should this be part of the automated bump process?

Reply:
i think it could be added to the hermit PR as both require a snapshot build before merging.